src/server/rdb_load.cc

```diff
@@ -203,6 +203,25 @@ int ziplistPairsConvertAndValidateIntegrity(const uint8_t* zl, size_t size, unsi
   return ret;
 }
 
+static string ModuleTypeName(uint64_t module_id) {
+  static const char ModuleNameSet[] =
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+      "abcdefghijklmnopqrstuvwxyz"
+      "0123456789-_";
+
+  char name[10];
+
+  name[9] = '\0';
+  char* p = name + 8;
+  module_id >>= 10;
+  for (int j = 0; j < 9; j++) {
```

Comment:
how did you check this? do you have rdb with modules?

Reply:
see here: https://github.com/redis/redis/blob/6.2/src/module.c#L4386

Reply:
I created manually with redis-stack. see attached.

[dump.rdb.gz](https://github.com/dragonflydb/dragonfly/files/14529277/dump.rdb.gz)

Reply:
it does not contain a module. it's module data, for example json entries are stored under module name `scdtype00`